src/native/external/libunwind/CMakeLists.txt

```diff
@@ -37,6 +37,11 @@ elseif ('$ENV{TARGET}' STREQUAL 'loongarch64-linux-gnu')
   set(arch loongarch64)
   add_definitions(-D__loongarch64)
   add_definitions(-D__linux__)
+elseif ('$ENV{TARGET}' STREQUAL 'riscv64-linux-gnu')
```

Comment:
This should be reverted. These files are not used by dotnet/runtime. We only use libunwind.cmake in parent directory without touching the upstream code.

Reply:
Thank you. I reverted.